cortex-m-pac/src/result.rs

```diff
@@ -0,0 +1,58 @@
+use core::fmt;
+
+/// Convenience alias for the [Result](core::result::Result) type for the library.
+pub type Result<T> = core::result::Result<T, Error>;
+
+/// Represents error variants for the library.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum Error {
```

Comment:
since `core::error::Error` is now stable it might make sense to `impl` it here? since you already have `impl core::fmt::Display` this should be straight-forward?

Reply:
done!